src/sentry/models/dsymfile.py

```diff
@@ -183,11 +189,37 @@ def dsym_type(self):
         ct = self.file.headers.get('Content-Type').lower()
         return KNOWN_DSYM_TYPES.get(ct, 'unknown')
 
+    @property
+    def supports_symcache(self):
+        # Only one that supports it so far.
+        return self.dsym_type == 'macho'
+
     def delete(self, *args, **kwargs):
         super(ProjectDSymFile, self).delete(*args, **kwargs)
         self.file.delete()
 
 
+class ProjectSymCacheFile(Model):
```

Comment:
This model is fine, but we need to add it to all the deletion stuff properly.

Reply:
Is there more deletion code? I added it to the place I found.

Reply:
Oh, you're right. All good.

Let's jsut get this model pulled out into it's own PR.